exercises/practice/clock/clock.awk

```diff
@@ -0,0 +1,4 @@
+BEGIN {
+    print "Implement this solution" > "/dev/stderr"
```

Comment:
If possible, I would use "Create this solution" or "Write this solution". The word "implement" means too many things, including having the meaning "Use this solution" which is confusing if this solution does not exist.

Reply:
We have too many people (like me) that uses a dictionary to figure out what people are saying, and having that many choices to choose from, and seeing people correctly use the word as it means, and in all the ways that this word means, is very confusing.  Better to have a "English as a not first language" friendly statement is probably a good thing to aspire to.

Reply:
We have this exact line in every awk exercise. I'm all for discussing a change, and the change ought to apply to all exercises. But I'd prefer that discussion happen in a dedicated "discuss stub" discussion and not as part of the "implement clock exercise" discussion.